### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by AmauryD** [miguelcobain/ember-yeti-table#461]
Are you interested to add a Typescript support (gjs -> gts) to this library ?

**Comment by miguelcobain** [comment]
Yes. I think Typescript is really beneficial, especially for addon code. 🙏

**Comment by cah-brian-gantzler** [comment]
Are you thinking of doing that in the V2 upgrade as well, or after the V2 version is merged?

**Comment by AmauryD** [comment]
> Are you thinking of doing that in the V2 upgrade as well, or after the V2 version is merged?

Because there's already a lot of changes in the V2 format PR. I will do this in another PR after the V2 format is merged.

**Comment by AmauryD** [comment]
Forgot about this issue. I'll take a look soon

**Comment by AmauryD** [comment]
Tracked by #468
